src/mtx3/utils/CommandBinaryBuffer.ts

```diff
@@ -43,5 +1313,145 @@ CommandBinaryBuffer.prototype.constructor = CommandBinaryBuffer;
+    };
+};
+
+CommandBinaryBuffer.prototype.setOperatorParameters = function (operatorParameters: IOperatorParameters) {
+    this.setUint32(operatorParameters.vpThreshold);
+    this.setUint32(operatorParameters.vThreshold);
+    this.setUint32(operatorParameters.ipThreshold);
+    this.setUint32(operatorParameters.pmaxThreshold0);
+    this.setUint32(operatorParameters.pmaxThreshold1);
+    this.setUint32(operatorParameters.pmaxThreshold2);
+    this.setUint32(operatorParameters.pmaxThreshold3);
+    this.setUint32(operatorParameters.rmaxThreshold0);
+    this.setUint32(operatorParameters.rmaxThreshold1);
+    this.setUint32(operatorParameters.rmaxThreshold2);
+    this.setUint32(operatorParameters.rmaxThreshold3);
+    this.setUint8(operatorParameters.tint);
+    this.setUint8(operatorParameters.calcPeriodDate);
+    this.setUint8(operatorParameters.timeoutDisplay);
+    this.setUint8(operatorParameters.timeoutScreen);
+    this.setUint32(bitSet.fromObject(displaySet1Mask, (operatorParameters.displaySet1 as unknown) as bitSet.TBooleanObject));
+    this.setUint32(bitSet.fromObject(displaySet2Mask, (operatorParameters.displaySet2 as unknown) as bitSet.TBooleanObject));
+    this.setUint32(bitSet.fromObject(displaySet3Mask, (operatorParameters.displaySet3 as unknown) as bitSet.TBooleanObject));
+    this.setUint32(bitSet.fromObject(relaySetMask, (operatorParameters.relaySet as unknown) as bitSet.TBooleanObject));
+    this.setUint8(setSpeedOptoPort(operatorParameters.speedOptoPort));
+    this.setUint8(operatorParameters.ten);
+    this.setUint8(operatorParameters.tu);
+    this.setUint8(operatorParameters.timeIntervalPowerOff);
+    this.setUint8(operatorParameters.reserved);
+    this.setUint8(operatorParameters.timeoutBadVAVB);
+    this.setUint8(operatorParameters.freqMax);
+    this.setUint8(operatorParameters.freqMin);
+    this.setUint8(operatorParameters.year);
+    this.setUint8(operatorParameters.month);
+    this.setUint8(operatorParameters.date);
+    this.setUint8(operatorParameters.energyDecimalPoint);
+    this.setUint16(operatorParameters.voltageTransformationRatioNumerator);
+    this.setUint16(operatorParameters.voltageTransformationRatioDenominator);
+    this.setUint16(operatorParameters.currentTransformationRatioNumerator);
+    this.setUint16(operatorParameters.currentTransformationRatioDenominator);
+    this.setUint8(bitSet.fromObject(typeMeterMask, (operatorParameters.typeMeter as unknown) as bitSet.TBooleanObject));
+    this.setUint16(operatorParameters.phMin);
+    this.setUint8(operatorParameters.timeoutIMax);
+    this.setUint8(operatorParameters.timeoutPMax);
+    this.setUint8(operatorParameters.timeoutCos);
+    this.setUint8(operatorParameters.pMaxDef);
+    this.setUint32(bitSet.fromObject(displaySet4Mask, (operatorParameters.displaySet4 as unknown) as bitSet.TBooleanObject));
+};
+
+
+export const getDefaultOperatorParameters = (): IOperatorParameters => (
```

Comment:
how about make it static method as in mtx1?

Reply:
fixed

Reply:
Probably not(